README.md

```diff
@@ -50,6 +50,17 @@ Install via any package management system, for example, paq:
 require paq { 'rktjmp/lush.nvim' }
 ```
 
+Via Lazy:
+
+```lua
+return {
+	"rktjmp/lush.nvim",
+
+    -- if you wish to actually use your own colorscheme:
+    -- { dir = '/absolute/path/to/colorscheme', lazy = true },
```

Comment:
- Fix indentation to use 4 spaces instead of tab
- Remove blank line
- Remove "actually" word